src/scripts/content/liveagent.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// ticket
+togglbutton.render(
+  '.ConversationDetailsView:not(.toggl)',
+  { observe: true },
+  function (
+    elem
+  ) {
+    const descriptionSelector = () => {
+      const ticketCode = $('.CodeLabel').textContent.trim();
```

Comment:
I know we wrap `window.$` to prevent jQuery usages:
https://github.com/toggl/toggl-button/blob/4dd4f0d9708b4f0bb3a109eb33d1020145007d1c/src/scripts/common.js#L8-L11

But can we use directly native functions instead? 🙏